src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -4,8 +4,9 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.Completion;
-using Microsoft.CodeAnalysis.Completion.Providers;
+using Microsoft.CodeAnalysis.Completion.Triggers;
```

Comment:
Should this be a separate namespace?  Seems so tightly tied to Completion that it should be in the same namespace.  Not sure.